core/platform/auth/firebase_auth_services_test.py

```
@@ -183,29 +183,41 @@ def create_session_cookie(
         self._uid_by_session_cookie[session_cookie] = claims['sub']
         return session_cookie
 
-    def create_user(
-        self, uid: str, email: Optional[str] = None, disabled: bool = False
-    ) -> str:
+    def create_user(  # pylint: disable=unused-argument
+        self,
+        uid: Optional[str] = None,
+        email: Optional[str] = None,
+        disabled: bool = False,
```

Ruff (0.15.15)

[warning] Boolean-typed positional argument in function definition

(FBT001)